test/js/api/order.js

```diff
@@ -270,4 +270,72 @@ describe("OrderAPI", function() {
             });
         });
     });
+

```

Comment:
Try to think of some more scenarios for this, such as subscribing and then unsubscribing, etc.